src/System.ComponentModel.TypeConverter/tests/MemberDescriptorTests.cs

```diff
@@ -9,6 +9,16 @@ namespace System.ComponentModel.Tests
 {
     public class MemberDescriptorTests
     {
+        static MemberDescriptorTests()
+        {
+            if(PlatformDetection.IsFullFramework)
+            {
+                // MethodDescriptor.Equals checks for this quirk and if it is set to true the behavior will be different to what we expect and the test will fail.
+                // This quirk is set to true by default on .NET versions earlier than 4.6.1 and Xunit runner was built against 4.5, so we override the quirk value to have the expected behavior.
+                AppContext.SetSwitch(@"Switch.System.MemberDescriptorEqualsReturnsFalseIfEquivalent", false);
```

Comment:
>AppContext.SetSwitch(@"Switch.System.MemberDescriptorEqualsReturnsFalseIfEquivalent", false); [](start = 16, length = 93)

could you assert on the quirk value inside the test? this will make it easier to catch failures if the quirk was not set correctly because of the caching.